### PR TITLE
Feat: add starkscan

### PR DESCRIPTION
**Pull request by 0xs34n**

### CODE REVIEW

File: data/starkscan.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  "description": "Next-generation StarkNet block explorer.",
+  "short_description": "StarkNet block explorer",
+  "tags": [],
+  "contracts": [],
+  "goerliContracts": [],
+  "audits": [],
+  "dotw": false,
+  "links": {
+    "website": "https://starkscan.co/?utm_source=dappland",
+    "careers": "jobs@aspect.co",
```
Comment (ismaeldm95) [inline data/starkscan.json:12]:
```suggestion
    "careers": "mailto:jobs@aspect.co",
```
Needs to include "mailto:" in order for it to work

File: data/starkscan.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  "twitterName": "StarkscanCo",
+  "teamInfo": {
+    "founded": "2022-09-28T00:00:00.000Z",
+    "anonymous": false
+  },
+  "tokens": [],
+  "media": {
+    "logoUrl": "https://pbs.twimg.com/profile_images/1558096338386141185/oT4BGZQr_400x400.jpg",
+    "bannerUrl": "https://pbs.twimg.com/profile_banners/1558095394558644225/1660314194/1500x500",
+    "previewUrl": "https://pbs.twimg.com/profile_banners/1558095394558644225/1660314194/1500x500",
```
Comment (ismaeldm95) [inline data/starkscan.json:30]: It looks like this in the homepage. Just checking if you guys have an image that fits better and doesn't crop the text.

Screenshot 2022-10-06 at 16 20 24

**Review by ismaeldm95 — CHANGES_REQUESTED** [review]
Thanks for adding StarkScan! super happy to see it included. Added a few comments

File: data/starkscan.json (hunk 2 above)
Comment (0xs34n) [reply data/starkscan.json:30]: This is the `previewUrl` i'm guessing? 

Can we try using the `logoUrl in there? See how that looks?

Comment (ismaeldm95) [reply data/starkscan.json]: looks way better! 
Screenshot 2022-10-10 at 12 16 36

**Comment by Aungko243877** [comment]
0xs34n:develop

File: data/starkscan.json (hunk 2 above)
Comment (Aungko243877) [reply data/starkscan.json]: hello

**Comment by Aungko243877** [comment]
0xs34n:develop

**Pull request merged**